lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -150,10 +150,7 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
 
             // if the over exist in the session layer
             if (spec->GetSpecifier() == SdfSpecifierOver) {
-                auto sessionLayer = prim.GetStage()->GetSessionLayer();
-                if (sessionLayer == spec->GetLayer()){
-                    layerDisplayName.append("[" + spec->GetLayer()->GetDisplayName() + "]" + ",");
-                }
+                layerDisplayName.append("[" + spec->GetLayer()->GetDisplayName() + "]" + ",");
```

Comment:
Remove redundant check to distinguish if the "over" specifier is in the session layer.